prepare.sh

```diff
@@ -61,7 +61,7 @@ tar zxf 'boost_1_69_0.tar.gz' -C 'contrib/boost-sdk' --strip-components=1
 echo "Building Docker Emscripten..."
 cd docker-deps/docker-emscripten
 docker build \
-  --platform linux/amd64 \
+  --platform linux/$(uname -m) \
```

Comment:
Please revert this change. It's needed to avoid issues with building under ARM (like in https://github.com/emscripten-core/emsdk/issues/671).